css/components.css

```diff
@@ -0,0 +1,93 @@
+.doc-container{
+    display: flex;
+    height: 100%;
+    font-family: 'Roboto', sans-serif;
+}
+
+.doc-content{
+    width: 100%;
+    padding: 3rem;
+    font-family: 'Nunito Sans', sans-serif; 
+}
+
+.heading{
+    font-size: 4rem;
+    font-weight: 500;
+}
+
+.intro-line{
+    font-size: 1.7rem;
+    margin-top: 1rem;
+    line-height: 30px;
+}
+
+.divider{
+    background-color: var(--bg-secondary-color);
+    margin-top: 1.5rem;
+    width: 100%;
+    height: 1px;
+}
+.sub-heading{
+    font-size:2rem ;
+    margin-top: 1rem;
+}
+
+.sub-head-content{
+    font-size:1.7rem;
+    line-height: 25px;
+    margin-top: 1rem;
+}
+
+.practical{
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
```

Comment:
why are you defining box-shadow props two times

Reply:
I have used multiple shadows here.